lib/services/config-api-service.js

```diff
@@ -39,7 +39,7 @@ function configApiServiceFactory(
 
     ConfigApiService.prototype.configSet = function(req) {
```

Comment:
Nevermind, this is doing the right thing.  I'd change the name though. `req` implies a request object to me.